File: Accenture/18.Prime/code.py
def checkPrime(n):
    if n <= 1:
        return False
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True
def print_primes_in_range(a, b):
    if a > 0 and b > 0 and a < b: 
        for num in range(a, b + 1):
            if checkPrime(num):
                print(num, end=' ')
a = int(input())
b = int(input())
print_primes_in_range(a, b)

File: Cognizant/1.Prime/code.cpp
#include <iostream>
#include <cmath>
using namespace std;
bool checkPrime(int n) {
    if (n <= 1)
        return false;
    for (int i = 2; i <= sqrt(n); i++) {
        if (n % i == 0)
            return false;}
    return true;}
void printPrimesInRange(int a, int b) {
    if (a > 0 && b > 0 && a < b) {
        for (int num = a; num <= b; num++) {
            if (checkPrime(num)) {
                cout << num << " ";
            }}
    } else {
        cout << "Invalid input range" << endl;
    }}
int main() {
    int a, b;
    cin >> a >> b;
    printPrimesInRange(a, b);
    return 0;}
